hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/RatisUtil.java

```diff
@@ -149,7 +149,8 @@ private static void setRaftRetryCacheProperties(
   private static void setRaftSnapshotProperties(
       final RaftProperties properties, final SCMHAConfiguration conf) {
     Snapshot.setAutoTriggerEnabled(properties, true);
-    Snapshot.setAutoTriggerThreshold(properties, 400000);
+    Snapshot.setAutoTriggerThreshold(properties,
+        conf.getRatisSnapshotThreshold());
```

Comment:
This is useful for both writing unit tests for snapshot functionality and allow users to set their own threshold